src/platform/windows.rs

```diff
@@ -104,6 +105,36 @@ impl MonitorHandleExtWindows for MonitorHandle {
     }
 }
 
+/// Additional methods on `OsSpecificWindowEvent` that are specific to Windows.
+pub trait OsSpecificWindowEventExtWindows {
```

Comment:
I'd rather not expose extensions.

Here is my proposal:

In `src/platform_impl/windows/mod.rs` we say `pub type OsSpecificWindowEvent = platform::windows::OsSpecificWindowEvent`.

We then define `OsSpecific(OsSpecificWindowEvent),` as `OsSpecific(platform_impl::OsSpecificWindowEvent),` and get rid of the `OsSpecificWindowEvent` in `src/events.rs`.

In this file we can then define `OsSpecificWindowEvent` as something like following (forgive the fake types):
```
pub enum Command {
    Menu(id),
    Accelerator(id),
    Control(code, id, handle),
}

pub enum OsSpecificWindowEvent {
    Command(Command),
    Notify(id, NMHDR_ptr)
}
```

Reply:
Hi, this will introduce quite a few concepts that `winit` itself consider out of scope(menus, controls, etc), and will grow more and more(and introduce lots of breaking change) whenever downstream crates has new requirements, like handling control colors and owner draws etc. Is it ok to go this way?

Reply:
Just for your interest: I've just started https://github.com/crlf0710/winit-featurepack to add os-specific missing features for `winit` for those functionality excluded.

Reply:
Ah, I wasn't exactly sure what those two commands were for. If that's the case, instead of `
pub enum OsSpecificWindowEvent`, we can have this:
```
pub struct OsSpecificWindowEvent {
    message: UINT,
    wparam: WPARAM,
    lparam: LPARAM,
}
```

Reply:
ok